crates/uv-client/src/registry_client.rs

```diff
@@ -363,9 +372,20 @@ impl RegistryClient {
                         let _permit = download_concurrency.acquire().await;
                         match index.format {
                             IndexFormat::Simple => {
-                                let metadata = self
-                                    .simple_single_index(package_name, index.url, capabilities)
-                                    .await?;
+                                let status_code_strategy =
+                                    IndexStatusCodeStrategy::ignore_authentication_error_codes();
```

Comment:
Why is this not `status_code_strategy_for`?

Reply:
I've added a comment. For unsafe index strategies, we ignore authentication failures and just find whatever matches we can across the indexes.